Lombiq.ContentEditors/Tests/Lombiq.ContentEditors.Tests.UI/Extensions/BasicAsyncEditorTestingUITestContextExtensions.cs

```diff
@@ -0,0 +1,23 @@
+using System.Threading.Tasks;
+using Lombiq.Tests.UI.Extensions;
+using Lombiq.Tests.UI.Services;
+using OpenQA.Selenium;
+
+namespace Lombiq.ContentEditors.Tests.UI.Extensions;
+
+public static class BasicAsyncEditorTestingUITestContextExtensions
+{
+    public static Task EnableContentEditorsSamplesFeatureAsync(this UITestContext context) =>
+        context.EnableFeatureDirectlyAsync("Lombiq.ContentEditors.Samples");
+
+    public static async Task<UITestContext> TestDemoAsyncEditorLoadOnAdminAsync(this UITestContext context)
+    {
+        await context.GoToRelativeUrlAsync("/Admin/ContentItemAsyncEditor/EmployeeAsyncEditorProvider/Employee");
+
+        context.Exists(By.XPath("//label[text()='Name']"));

```

Comment:
```suggestion
        context.Exists(By.XPath("//label[. = 'Name']"));
```
Don't trust the `text()`, too fragile. Same below.

Reply:
I meant these 2 by "same below": https://github.com/Lombiq/Orchard-Content-Editors/pull/21/files#diff-acf61e5846929ac929a5b7dbae348589446fb5b2f6afaa352b5f83c5f8335d99R18-R19